src/emmy/expression/analyze.cljc

```diff
@@ -81,25 +82,53 @@
       :else (compare v w))))
 
 (defn monotonic-symbol-generator
-  "Returns a function which generates a sequence of symbols with the given
-  `prefix` with the property that later symbols will sort after earlier symbols.
-
-  This is important for the stability of the simplifier. (If we just used
-  `clojure.core/gensym`, then a temporary symbol like `G__1000` will sort
-  earlier than `G__999`. This will trigger errors at unpredictable times,
-  whenever `clojure.core/gensym` returns two symbols that cross an
-  order-of-magnitude boundary.)"
-  [prefix]
-  (let [count (atom -1)]
```

Comment:
IIRC this `-1` was why the old version had symbols starting at 0... but probably not worth changing and making more work to migrate the tests, when you've done this so many times!

Reply:
I thought about this and there were just fewer tests to hack on with it done this way.